<?xml version='1.0'?>
<ns2:test-suite xmlns:ns2='urn:model.allure.qatools.yandex.ru' start='1744694105181' stop='1744694106277'>
    <name>API_Invalid Login Validation:</name>
    <title>API_Invalid Login Validation:</title>
    <test-cases>
        <test-case start='1744694105182' status='failed' stop='1744694105899'>
            <name>Login attempt without password should fail {"email":"sdfdf@test.in","errorMessage":"Missing password"}</name>
            <title>Login attempt without password should fail {"email":"sdfdf@test.in","errorMessage":"Missing password"}</title>
            <labels>
                <label name='language' value='javascript'/>
                <label name='framework' value='codeceptjs'/>
            </labels>
            <parameters/>
            <steps>
                <step start='1744694105540' status='passed' stop='1744694105668'>
                    <name>I resize window 1900, 1100</name>
                    <title>I resize window 1900, 1100</title>
                    <attachments/>
                    <steps/>
                </step>
                <step start='1744694105669' status='passed' stop='1744694105779'>
                    <name>I resize window 1900, 1100</name>
                    <title>I resize window 1900, 1100</title>
                    <attachments/>
                    <steps/>
                </step>
                <step start='1744694105779' status='failed' stop='1744694105899'>
                    <name>When I try to login with email "sdfdf@test.in" and no password ""</name>
                    <title>When I try to login with email "sdfdf@test.in" and no password ""</title>
                    <attachments/>
                    <steps>
                        <step start='1744694105779' status='failed' stop='1744694105899'>
                            <name>I send post request "api/login", {"email":"sdfdf@test.in"}</name>
                            <title>I send post request "api/login", {"email":"sdfdf@test.in"}</title>
                            <attachments/>
                            <steps/>
                        </step>
                    </steps>
                </step>
            </steps>
            <attachments>
                <attachment title='Main session - Last Seen Screenshot' source='0ee32b43-f116-444f-866d-c0bdcd154056-attachment.png' type='image/png' size='8467'/>
            </attachments>
            <failure>
                <message>unable to get local issuer certificate</message>
                <stack-trace>Error: unable to get local issuer certificate
                    at Function.AxiosError.from (C:\Users\reddym1\Documents\TOOLS-G\PlaywrightTask\Codecept_Sacedemo_M2\node_modules\axios\lib\core\AxiosError.js:92:14)
                    at RedirectableRequest.handleRequestError (C:\Users\reddym1\Documents\TOOLS-G\PlaywrightTask\Codecept_Sacedemo_M2\node_modules\axios\lib\adapters\http.js:620:25)
                    at RedirectableRequest.emit (node:events:520:28)
                    at RedirectableRequest.emit (node:domain:488:12)
                    at ClientRequest.eventHandlers.&lt;computed> (C:\Users\reddym1\Documents\TOOLS-G\PlaywrightTask\Codecept_Sacedemo_M2\node_modules\follow-redirects\index.js:49:24)
                    at ClientRequest.emit (node:events:520:28)
                    at ClientRequest.emit (node:domain:488:12)
                    at emitErrorEvent (node:_http_client:103:11)
                    at TLSSocket.socketErrorListener (node:_http_client:506:5)
                    at TLSSocket.emit (node:events:520:28)
                    at Axios.request (C:\Users\reddym1\Documents\TOOLS-G\PlaywrightTask\Codecept_Sacedemo_M2\node_modules\axios\lib\core\Axios.js:45:41)
                    at processTicksAndRejections (node:internal/process/task_queues:95:5)
                    at async REST._executeRequest (C:\Users\reddym1\Documents\TOOLS-G\PlaywrightTask\Codecept_Sacedemo_M2\node_modules\codeceptjs\lib\helper\REST.js:229:18)
                    at async C:\Users\reddym1\Documents\TOOLS-G\PlaywrightTask\Codecept_Sacedemo_M2\tests\api\step_definitions\Invalidlogin.steps.ts:8:14
                    at async runSteps (C:\Users\reddym1\Documents\TOOLS-G\PlaywrightTask\Codecept_Sacedemo_M2\node_modules\codeceptjs\lib\mocha\gherkin.js:85:9)</stack-trace>
            </failure>
        </test-case>
    </test-cases>
</ns2:test-suite>